src/mips/psyqo/hardware/cdrom.hh

```diff
@@ -2,7 +2,7 @@
 
 MIT License
 
-Copyright (c) 2023 PCSX-Redux authors
+Copyright (c) 2025 PCSX-Redux authors
```

Comment:
Don't bump copyrights unless there's a fundamental rewrite of the file.

Reply:
Ah, I thought it was just on updates

Reply:
I've been following the guidance of an OSS lawyer from a decade ago, basically, who said that copyright date of a single file starts when the file is first created, and the date should only bump when the file is fundamentally being rewritten. The reasoning is this is to know when the copyright actually expires, ala Mickey Mouse.